polling_stations/apps/api/tests/test_address.py

```diff
@@ -202,3 +205,60 @@ def test_bad_slug(self):
         )
 
         self.assertEqual(404, response.status_code)
+
+
+class GetEEWrapperTest(TestCase):
```

Comment:
Previously, the `get_ee_wrapper` methods of the postcode and API endpoints were not under test at all because we were just mocking out the whole method in the main tests to avoid having to mock individual API calls out to EE. This wasn't a huge issue when this method wasn't doing very much (just constructing a class and returning it), but there is now enough going on in those methods that they need to be under test. These tests are perhaps not perfect, but they do give use a starting point and cover the cases I am introducing here.